### ISSUE TITLE
Editor: Occasionally files are not written when saved.

### ISSUE THREAD

**Issue by MrYsLab** [dddomodossola/remi#133]

**Comment by MrYsLab** [comment]
The next time it happens I will upload the debug console output.

**Comment by KenT2** [comment]
Does this happen when running the server on Windows. I had a similar problem. Solved it by detecting an IOError exception and trying again.

**Comment by MrYsLab** [comment]
@KenT2 I am not sure if the question is directed towards me, but I develop on Linux, so I don't know if this happens on Windows or not.

**Comment by KenT2** [comment]
@MrYsLab Yes it was. I had a similar problem. When using Python files saved every time on Linux but failed occasionally on Windows.
